src/components/homepage/utils.tsx

```diff
@@ -0,0 +1,48 @@
+export type ChoosingCardsItemsType = {
```

Comment:
No need for this file to be a .tsx file as you're not handling any jsx code ;)

Reply:
Ah yes !